### PR TITLE
feat: handle mint op for user deposit tx

### PR DESCRIPTION
**Pull request by GeekArthur**
Signed-off-by: Jingfu Wang <jingfu.wang@coinbase.com>

### CODE REVIEW

File: pkg/handlers/utils.go [hunk 1]
```diff
@@ -1,9 +1,17 @@
 package handlers
 
 import (
+	"strings"
+
 	"github.com/coinbase/rosetta-sdk-go/types"
 )
 
+const (
+	proxyContractPrefix          = "0x42"
```
Comment (jingweicb) [inline pkg/handlers/utils.go]: all the address with "0x42", do we wan to add more 0000 to avoid potential personal accs?

Comment (GeekArthur) [reply pkg/handlers/utils.go]: Actually I think I can use `proxyContractFilter` (the longer one)

Comment (GeekArthur) [reply pkg/handlers/utils.go]: This is only for short circuit evaluation, so longer one is good 👍

Comment (GeekArthur) [reply pkg/handlers/utils.go]: Done

**Review by jingweicb — APPROVED** [review]

**Pull request merged**